models/modelling/commissioning/consolidated/int_ecds_consolidated.sql

```
@@ -0,0 +1,327 @@
+WHEN "ATTENDANCE_LOCATION_SITE" = 'RYX02' AND "ATTENDANCE_LOCATION_DEPARTMENT_TYPE" IN ('03', '04') THEN 'WIC_SOHO'			--Soho Walk-In-Centre 
+WHEN "ATTENDANCE_LOCATION_SITE" = 'RYX01' AND "ATTENDANCE_LOCATION_DEPARTMENT_TYPE" IN ('03', '04') THEN 'WIC_STCharles'	--St Charles Walk-In Centre 
+WHEN "ATTENDANCE_LOCATION_SITE" = 'RAS02' AND "ATTENDANCE_LOCATION_DEPARTMENT_TYPE" = '03' THEN 'MIU_MV'					--Mount Vernon Hospital 
+WHEN "ATTENDANCE_LOCATION_SITE" = 'RYJ02' AND "ATTENDANCE_LOCATION_DEPARTMENT_TYPE" = '03' THEN 'UTC_CX'					--Charing Cross UTC
+
+--NOT CURRENTLY SUBMITTING: 
+WHEN "ATTENDANCE_LOCATION_SITE" = 'RYJ03' AND "ATTENDANCE_LOCATION_DEPARTMENT_TYPE" = '03' THEN 'UTC_HH'					--Hammersmith Hospital UTC 
+ELSE NULL 
+END																			AS VARCHAR(50))	AS zBusinessRule
+
+
+FROM {{ref('raw_sus_ae_emergency_care')}} att 
```

SQLFluff (4.2.2)

[error] Too many consecutive blank lines.

(LT15)